# scRNAseq

# Step 1a: Set up working environments
* Install software / tools on our server
* Install R package for data process 

# Step 1b: Download data set 
* Single cell data from tumor with virus infection vs. no-infection

# Step 2a: Practice data process with example data 
* Guidline: https://www.bioinformatics.babraham.ac.uk/training/10XRNASeq/Introduction.pdf
* R script for example: https://github.com/bishwaG/Single-cell-RNA-seq-data-analysis-2018

* Pre-process using 10X cell ranger : https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/tutorial_ov

* scRNAseq data quality control check

* cluster analysis and cell type identification 

* review for single cell data analysis : https://www.biorxiv.org/content/biorxiv/early/2018/10/08/433102.full.pdf

# Step 2b: Run pipeline with real data 
*  Setup workflow:  
** Using R package provided by the paper: https://arc85.github.io/dragonsc/articles/dragonsc.html
** Follow the instruction to reproduce the simulation data. 

*  Processed data from Cillo et.al   /public/home/jinxu/Projects/HPV_HNSCC
** Modify the script to read real data and do the following analysis 



